<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Sort">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinRow">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinColumn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Row">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Строки</string>
    </property>
   </widget>
   <widget class="QLabel" name="Column">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Столбцы</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Sort_of_table">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>210</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Bubble sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quicksort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comb sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gnome sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bogosort (Обезьянья)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировки:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRand">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
